pkg/engine/api/ruleresponse.go

```diff
@@ -143,6 +150,10 @@ func (r *RuleResponse) Exceptions() []GenericException {
 func (r *RuleResponse) ValidatingAdmissionPolicyBinding() *admissionregistrationv1.ValidatingAdmissionPolicyBinding {
 	return r.binding
 }
+func (r RuleResponse) WithMutatingBinding(binding *admissionregistrationv1alpha1.MutatingAdmissionPolicyBinding) *RuleResponse {
+	r.mutatingBinding = binding
+	return &r
+}
```

Comment:
Same comment.

Reply:
Can you remove that for now? It will be needed in case of generating reports for MAPs, but this PR is for the CLI.